### PR TITLE
Update readme with basic usage instructions for install and filter script 

### PR DESCRIPTION
**Pull request by rlskoeser**
addresses #64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,82 @@
-# `corppa`  PPA full-text corpus utilities
+# corppa
+
+This repository is research software developed as part of the [Ends of Prosody](https://cdh.princeton.edu/projects/the-ends-of-prosody/), which is associated with the [Princeton Prosody Archive](https://prosody.princeton.edu/) (PPA). This software is particularly focused on research and work related to PPA full-text and page image corpora.
+
+> [!WARNING]
```
Comment (rlskoeser) [inline README.md:5]: Trying out the format I found here https://github.com/orgs/community/discussions/16925

File: README.md [hunk 2]
```diff
@@ -1,25 +1,82 @@
...
+Use pip to install as a python package directly from GitHub.  Use a branch or tag name, e.g. `@develop` or `@0.1` if you need to install a specific version.
+
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git#egg=corppa
+```
+or
+```sh
+pip install git+https://github.com/Princeton-CDH/ppa-nlp.git@v0.1#egg=corppa
+```
+
```
Comment (rlskoeser) [inline README.md:21]: Anything to add about conda install?

(I didn't actually test these and probably should)

File: README.md [hunk 3]
```diff
@@ -1,25 +1,82 @@
...
+Installing `corppa` currently provides access to two command line scripts, for filtering a PPA page-level corpus or for generating OCR text for images using Google Vision API. These can be run as `corppa-filter-corpus` and `corppa-ocr` respectively.
+
+#### Filtering PPA page-text corpus
+
+The PPA page-level text corpus is shared as a json lines (`.jsonl`) file, which may or may not be compressed (e.g., `.jsonl.gz`). It's often useful to filter the full corpus to a subset of pages for a specific task, e.g. to analyze content from specific volumes or select particular pages for annotation.
+
+To create a subset corpus with _all pages_ for a set of specific volumes, create a text file with a list of **PPA work identifiers**, one id per line, and then run the filter script with the input file, desired output file, and path to id file.
+
+```sh
+corppa-filter-corpus ppa_pages.jsonl my_subset.jsonl --idfile my_ids.txt
```
Comment (rlskoeser) [inline README.md:33]: Do you think we could shorten to just corppa-filter ? It's such a long script name

File: README.md (hunk 1 above)
Comment (laurejt) [reply README.md:5]: Is there a place we should document this for future reference?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,82 @@
...
+```
+
+> [!NOTE]
+> **PPA work identifiers** are based on source identifiers, i.e., the identifier from the original source (HathiTrust, Gale/ECCO, EEBO-TCP). In most cases the work identifier and the source identifier are the same, but _if you are working with any excerpted content the work id is NOT the same as the source identifier_. Excerpt ids are based on the combination of source identifier and the first original page included in the excerpt. In some cases PPA contains multiple excerpts from the same source, so this provides guaranteed unique work ids.
+
+To create a subset of _specific pages_ from specific volumes, create a CSV file that includes `work_id` and `page_num`, and pass that to the filter script with the `--pg-file` option:
```
Comment (laurejt) [inline README.md:39]:
```suggestion
To create a subset of _specific pages_ from specific volumes, create a CSV file that includes fields `work_id` and `page_num`, and pass that to the filter script with the `--pg-file` option:
```

**Review by laurejt — APPROVED** [review]

**Comment by laurejt** [comment]
Once this is merged, I'll add some documentation on the ocr script.

File: README.md (hunk 1 above)
Comment (rlskoeser) [reply README.md:5]: the syntax? maybe we could add a link to some team documentation somewhere...

Comment (laurejt) [reply README.md:5]: Yeah, that sounds good to me. This seems like a nice feature to use.

**Pull request merged**